scripts/check_for_docs_deps_changes

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+CHANGED_FILES=$(git diff HEAD origin/develop --name-only)
+printf '%s\n' "[CHANGED FILES]" "${CHANGED_FILES[@]}" ""
+
+pip install -e . > /dev/null 2>&1 # Necessary to get GE helper methods working for `trace_docs_deps.py`
+
```

Comment:
@alexsherstinsky we actually need to install GE if we want to use the `file_relative_path()` helper method in our dependencies script.

I don't think this is worth it as the remainder of script works as intended without any external dependencies.